tools/whois.init

```diff
@@ -167,12 +167,20 @@ case "$1" in
                         exit 1
                 fi
                 ;;
+        disable)
```

Comment:
There's a naming collision with the existing ```disable``` in the internal init script. If the intention is to replace that altogether then that's OK.

Reply:
This script is only used for when running whois locally. I am not sure I know the internal init script you refer to, but I do not think there should be any collision or confusion...?

Reply:
I mean the DB team's ```/export/init.d/whois``` script, will these changes be synced there?

Reply:
Nope. It won't. This script is so different from the ones in `/export/init.d/whois` I can only imagined it diverged a long time ago and it is only used when running locally. 

I mainly updated it just to make sure that using the jmxconsole via bash script works...I would still have to update the `/export/init.d/whois` to incorporate the new ability to disable/enable via jmx call. That would probably be a different PR on db-operational first, before changing it on staging server

Reply:
OK! Thanks for explaining.